src/modules/NetworkLogger.ts

```diff
@@ -2,14 +2,28 @@ import type { RequestHandler } from '@apollo/client';
 
 import InstabugConstants from '../utils/InstabugConstants';
 import xhr, { NetworkData, ProgressCallback } from '../utils/XhrNetworkInterceptor';
-import { reportNetworkLog, isContentTypeNotAllowed } from '../utils/InstabugUtils';
+import { isContentTypeNotAllowed, reportNetworkLog } from '../utils/InstabugUtils';
+import { NativeModules } from 'react-native';
 
 export type { NetworkData };
 
 export type NetworkDataObfuscationHandler = (data: NetworkData) => Promise<NetworkData>;
 let _networkDataObfuscationHandler: NetworkDataObfuscationHandler | null | undefined;
 let _requestFilterExpression = 'false';
 
+const filterURL = (url: string): string => {
+  const [protocol, rest] = url.split('://');
+  return (rest || protocol).split('/')[0];
+};
+
+const getDevServerURL = (): string | null => {
+  const { scriptURL } = NativeModules.SourceCode;
```

Comment:
Is there a higher level API that we can use instead of using `NativeModules.SourceCode`? If not, I believe we need to check first if `SourceCode` is not `null` or `undefined` before trying to access `scriptURL` on it.

Reply:
I didn't find any other API, but I added a condition to check first.